src/frontend/mod.rs

```diff
@@ -259,6 +259,15 @@ impl MainPodBuilder {
                 let [array, index, value] = op.1.try_into().unwrap(); // TODO: Error handling
                 Operation(Native(ContainsFromEntries), vec![array, index, value], op.2)
             }
+            Native(GtEqFromEntries) => {
```

Comment:
Can you add a new line to the function documentation summarizing these transformations?